src/translations/indexMapper.js

```diff
@@ -162,20 +162,20 @@ class IndexMapper {
   }
 
   /**
-   * Execute batch operations with updating cache when necessary. As effect, wrapped operations will be executed and
-   * cache will be updated at most once (cache is updated only when any cached index has been changed).
-   *
-   * @param {Function} wrappedOperations Batched operations wrapped in a function.
+   * Suspends the cache update for this map. The method is helpful to group multiple
+   * operations, which affects the cache. In this case, the cache will be updated once after
+   * calling the "resumeOperations" method.
```

Comment:
Ok, I see that here "resumeOperations" is also inside double quotes, is there a reason for that? :) I mean, why not `code`.

Reply:
Sure! Done within https://github.com/handsontable/handsontable/pull/7406/commits/e2e75b2102da96583a572fa2c0416c7475f0f89f.